src/components/header/template.njk

```diff
@@ -93,6 +94,9 @@
     {{ govukPhaseBanner(params.phaseBanner) }}
   </div>
 {% endif %}
+{% if params.serviceNavigation %}
+    {{ govukServiceNavigation(params.serviceNavigation) }}
+{% endif %}
```

Comment:
just realised that I placed this in the wrong spot, service navigation should go above phase banner according to gov.uk design system guidance 

https://design-system.service.gov.uk/components/phase-banner/